opendistro-elasticsearch-security.release-notes

```diff
@@ -1,4 +1,9 @@
-## 2020-03-05, Version 0.10.1.1 (Current)
+## 2020-03-31, Version 0.10.1.2 (Current)
+
+- Fixed bug which caused user to lose roles on account password update
+- Performance improvement by implementing faster version of implies type perm
```

Comment:
Can't see performance improvement changes as part of this PR

Reply:
It is present in the branch but a release was not made. 
It will be going out with this release.

Reply:
Thanks for clarifying